src/helpers/fetchAbis.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+const CONTRACTS = ['BatchExchange']
+const { abi } = require('@gnosis.pm/dex-contracts/build/contracts/BatchExchange.json')
+var ABI_DIR = path.join(__dirname, '../contracts/abi')
+
+if (!fs.existsSync(ABI_DIR)) {
+  fs.mkdirSync(ABI_DIR)
+}
+for (const contractName of CONTRACTS) {
```

Comment:
In a loop, shouldn't this `const contractName` be a `let`?

Reply:
Not sure, is it more efficient with the memory or sth? const will leave within the scope of that loop afaik, furthermore, it tells that within the body of the loop I don't anyone to change it's value.

Reply:
I thought this would be a syntactic error since `contractName` would be updated on every iteration of the loop. If this works then I'm just wrong.